keyboards/lily58/keymaps/muppetjones/rules.mk

```diff
@@ -0,0 +1,41 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = no        # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+OLED_DRIVER_ENABLE= yes     # OLED display
+WPM_ENABLE = yes
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+TAP_DANCE_ENABLE = yes
+
+LTO_ENABLE = yes
```

Comment:
```suggestion
WPM_ENABLE = yes
TAP_DANCE_ENABLE = yes
LTO_ENABLE = yes
```

Reply:
Leaving this was intentional, and as comment lines aren't included at compile time, it doesn't hurt to leave them in a personal keymap.

This is also from the default lily58 keymap.

Reply:
The idea here is to reduce duplication. Since most of this stuff is the same as the keyboard-level rules.mk, it can be removed, leaving the keymap-level rules.mk containing only the changes vs. keyboard-level.